doc/_includes/inverse.rst

```diff
@@ -184,32 +192,63 @@ regularizing the channel covariance matrix.
 .. _mne_solution:
 
 Computation of the solution
----------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The most straightforward approach to calculate the MNE is to employ expression
 for the original or whitened inverse operator directly. However, for
 computational convenience we prefer to take another route, which employs the
 singular-value decomposition (SVD) of the matrix
 
-.. math::    A = \tilde{G} R^{^1/_2} = U \Lambda V^T
+.. math::
+   :label: inv_a
+
+    A &= \tilde{G} R^{^1/_2} \\
+      &= U \Lambda V^T
 
 where the superscript :math:`^1/_2` indicates a square root of :math:`R`. For a
 diagonal matrix, one simply takes the square root of :math:`R` while in the
 more general case one can use the Cholesky factorization :math:`R = R_C R_C^T`
 and thus :math:`R^{^1/_2} = R_C`.
 
-With the above SVD it is easy to show that
+Combining the SVD from :eq:`inv_a` with the inverse equation :eq:`inv_m` it is
+easy to show that
 
-.. math::    \tilde{M} = R^{^1/_2} V \Gamma U^T
+.. math::
+   :label: inv_m_tilde_svd
 
-where the elements of the diagonal matrix :math:`\Gamma` are
+    \tilde{M} &= R \tilde{G}^T (\tilde{G} R \tilde{G}^T + I)^{-1} \\
+              &= R^{^1/_2} A^T (A A^T + I)^{-1} \\
+              &= R^{^1/_2} V \Lambda U^T (U \Lambda V^T V \Lambda U^T + I)^{-1} \\
+              &= R^{^1/_2} V \Lambda U^T (U (\Lambda^2 + I) U^T)^{-1}
+              &= R^{^1/_2} V \Lambda U^T U (\Lambda^2 + I)^{-1} U^T
+              &= R^{^1/_2} V \Gamma U^T
 
-.. math::    \gamma_k = \frac{1}{\lambda_k} \frac{\lambda_k^2}{\lambda_k^2 + \lambda^2}\ .
+where the elements of the diagonal matrix :math:`\Gamma` are simply
 
-With :math:`w(t) = U^T C^{-^1/_2} x(t)` the expression for the expected current
-is
+.. `reginv` in our code:
 
-.. math::    \hat{j}(t) = R^C V \Gamma w(t) = \sum_k {\bar{v_k} \gamma_k w_k(t)}\ ,
+.. math::
+   :label: inv_gamma_k
+
+    \gamma_k &= \frac{1}{\lambda_k} \frac{\lambda_k^2}{\lambda_k^2 + \lambda^2} \\
+             &= \frac{\lambda_k}{\lambda_k^2 + \lambda^2}\ .
+
+From our expected current equation and :eq:`inv_j_hat_t` and our whitened
+measurement equation :eq:`inv_tilde_x_t`, if we take
+
+.. math::
+   :label: inv_w_t
+
+    w(t) &= U^T \tilde{x}(t) \\
+         &= U^T C^{-^1/_2} x(t)\ ,
+
+we can see that the expression for the expected current is just
+
+.. math::
+   :label: inv_j_hat_t_svd
+
+    \hat{j}(t) &= R^C V \Gamma w(t) \\
```

Comment:
I would use R^{^1/_2} as it's what is used above. Not R^C

Reply:
Yes that was already there and I wasnt sure. Feel free to push that and the other fix as I'm away for some hours. Will get to it later if you don't